.azure-pipelines/dependency-check.yml

```diff
@@ -4,8 +4,8 @@ steps:
 
     pip3 install natsort
 
-    CHECK_RESULT=$(python3 ./.azure-pipelines/dependency_check/dependency_check.py tests)
-    if [[ "$CHECK_RESULT" == "True" ]]; then
+    python3 ./.azure-pipelines/dependency_check/dependency_check.py tests
+    if [[ $? -ne 0 ]]; then
       echo "##vso[task.complete result=Failed;]Condition check failed."
```

Comment:
Why set variable `result` here?

Reply:
As I searched, ##vso[task.complete result=Failed;] is used to explicitly mark a task as failed. And exit 1 is a double confirm to fail the pipeline.